tests/testthat/test-pkg_examples.R

```diff
@@ -1,28 +1,31 @@
+library(RcppProgress)
```

Comment:
a test must not load the package it tests.

Reply:
it is already loaded in https://github.com/kforner/rcpp_progress/blob/master/tests/testthat.R

Reply:
didn't know that, thanks!